test/e2e/webdriver/driver.js

```diff
@@ -1123,6 +1123,10 @@ class Driver {
       'Failed to load resource: the server responded with a status of 429',
       // 4Byte
       'Failed to load resource: the server responded with a status of 502 (Bad Gateway)',
+      // Event fragment not found and SES Unhandled errors (issue #26089)
+      'Event fragment with id',
+      'SES_UNHANDLED_REJECTION',
+      'at Object.updateEventFragment',
     ]);
```

Comment:
we need to add the errors into the default ignored list, since the errors can appear to any tests which perform a transaction (which are several). Despite the other tests not being so flaky as the Send ETH to a 40 hexadecimal, they still can have ocurrences, like this one https://app.circleci.com/pipelines/github/MetaMask/metamask-extension/93255/workflows/736b665d-7f70-4083-bff5-1aaecaa38e49/jobs/3472270/tests